src/components/Renderer/index.tsx

```diff
@@ -70,6 +66,40 @@ export const getWidget = (
 	return typeWidgets[valueType] ?? typeWidgets.default;
 };
 
+export interface WidgetProps {
+	value?: Value;
+	schema?: JSONSchema;
+	extraContext?: object | undefined;
+	uiSchema?: UiSchema;
+	uiSchemaWidget?: UiSchema['ui:widget'];
+	extraFormats?: Format[];
+}
+
+export const Widget = ({
```

Comment:
Not a strong opinion, but now that we exporting it, how making this a bit more specific?
How about `ValueWidget` or `RendererWidget`?